source/copilot-schema.txt

```diff
@@ -0,0 +1,63 @@
+.. _vsce-copilot-schema:
+
+===============
+/schema Command
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The ``/schema`` command provides information about your collection
+schema. The |copilot| accesses your connected MongoDB deployment to
+provide schema information in the Copilot generated response. 
+
+Examples
+--------
+
+Provide a Sample Document
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider the ``users`` collection in the `Mflix Sample Database
```

Comment:
I'd rewrite this to not use "Consider" because what is reader being asked to consider?

Reply:
Done!